tests/unit/test_utils.py

```diff
@@ -61,7 +64,20 @@ def test_compare_uni_out_trace(self):
 
     def test_map_dl_vlan(self):
         """Test map_dl_vlan"""
-        cases = {0: None, 'untagged': None, 'any': 1, '4096/4096': 1, 10: 10}
+        cases = {0: None, "untagged": None, "any": 1, "4096/4096": 1, 10: 10}
         for value, mapped in cases.items():
             result = map_dl_vlan(value)
             assert result == mapped
+
+    def test_get_vlan_tags_and_masks(self):
```

Comment:
It'd be great if you could also parametrize a unit test covering these edges:

- when `start == end` -> [(`start`, 4095)]
- when `start > end` -> []

Reading the code, I can tell it'll work but it'd be great to have this unit tested too.